samples/js/browser/avatar/js/basic.js

```diff
@@ -179,8 +179,19 @@ window.startSession = () => {
         alert('Please fill in the subscription key of your speech resource.')
         return
     }
+    const privateEndpointCheckBox = document.getElementById('enablePrivateEndpoint').checked
+    const privateEndpoint = document.getElementById('privateEndpoint').value.slice(8)
+    if( privateEndpointCheckBox && privateEndpoint === ''){
+        alert('Please fill in the Azure Speech endpoint.')
+        return
+    }
 
-    const speechSynthesisConfig = SpeechSDK.SpeechConfig.fromSubscription(cogSvcSubKey, cogSvcRegion)
+    let speechSynthesisConfig
+    if(privateEndpointCheckBox){
+        speechSynthesisConfig = SpeechSDK.SpeechConfig.fromEndpoint(new URL(`wss://${privateEndpoint}/tts/cognitiveservices/websocket/v1?enableTalkingAvatar=true`), cogSvcSubKey) 
+    }else {
```

Comment:
Regarding to space, please keep consistent coding style with existing code, this line should be like below:

    } else {